src/main/java/uk/co/evoco/webdriver/configuration/driver/ConfiguredChromeDriver.java

```diff
@@ -38,6 +43,32 @@ public WebDriver getLocalDriver() throws IOException {
      */
     public ChromeOptions getOptions() {
```

Comment:
Be great to tidy this up a little Bala - it seems a little long winded?

Reply:
changed to create the download folder in target folder

Reply:
That wasnt the point of this comment.  This method is long winded - using an iterator is fine, but theres a lot going on.  Can you consider refactoring it to be cleaner?